internal/graphql/resolver/performance.go

```diff
@@ -91,7 +91,7 @@ func (r *Resolver) fetchResults(ctx context.Context, provider models.Provider, s
 		return nil, handlers.ErrQueryGet("*profileID")
 	}
 
-	tokenString := ctx.Value("token").(string)
+	tokenString := ctx.Value(models.TokenCtxKey).(string)
```

Comment:
we use the previous "token" key for getting the tokenString from the context for other resolvers and providers for REST, so should this change be propagated to those functions too?

Reply:
`models.TokenCtxKey` is actually `"token"`